activities/PHQ-9-LORIS/phq9_page1.jsonld

```diff
@@ -0,0 +1,25 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/contexts/generic.jsonld",
+        "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/activities/PHQ-9-LORIS/phq9_context.jsonld"
+    ],
+    "@type": "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/schemas/LayoutElement.jsonld",
```

Comment:
@sanuann - i thought we were doing this through sub activities. is there a reason to use what @zaliqarosli is proposing relative to a sub activity?

Reply:
yes. we are doing this through sub-activities. @zaliqarosli you can check this out: https://github.com/ReproNim/schema-standardization/blob/master/activities/voice_task/multipart_voice_schema.jsonld#L5 
The idea is - an activity can contain sub-activities and/or items.

Reply:
are you guys proposing this for pages only or for all types of "layout elements" that we need? I am happy to define pages as multiparts but not sure how a table element would be covered by this.

what if a multi-part (a page) contains another multipart (a section inside that page)? how would we know the difference between a page and a section when rendering the instrument?

Reply:
@zaliqarosli the value to the key `inputType` is not restricted by the schema. I think you can use it to distinguish between a page and section.